Makefile

```diff
@@ -27,6 +27,7 @@ install:
 	-rm -rf dist
 	python setup.py bdist_wheel
 	pip install --upgrade dist/*.whl
+	$(MAKE) bindings
```

Comment:
Wild idea: can travis commit into the repo?  We should have a travis cronjob build the bindings, and then commit them if there's any changes. :)